static/app/views/insights/navigation.tsx

```diff
@@ -60,7 +62,9 @@ export default function InsightsNavigation({children}: InsightsNavigationProps)
             <SecondaryNav.Item to={`${baseUrl}/${MOBILE_LANDING_SUB_PATH}/`}>
               {MOBILE_SIDEBAR_LABEL}
             </SecondaryNav.Item>
-            <SecondaryNav.Item to={`${baseUrl}/${AI_LANDING_SUB_PATH}/`}>
+            <SecondaryNav.Item
+              to={`${baseUrl}/${AI_LANDING_SUB_PATH}/${MODULE_BASE_URLS.ai}/`}
```

Comment:
Nit, but I think we typically do
```suggestion
              to={`${baseUrl}/${AI_LANDING_SUB_PATH}/${MODULE_BASE_URLS.[ModuleName.Ai]}/`}
```

Reply:
Roger, thanks!